src/components/SkipLink/SkipLink.astro

```diff
@@ -0,0 +1,23 @@
+---
+import Button from '@components/Button/Button.astro';
+
+interface Props {
+    label?: string;
+    class?: string;
+    target?: string;
+}
+
+const { label = 'Skip to main content', target, class: propsClass, ...rest } = Astro.props;
+
+const classes = ['c-skip-link', propsClass];
+---
+
+<c-skip-link class:list={classes} {...rest} data-no-swup>
+    <Button href="#main">{label}</Button>
+</c-skip-link>
```

Comment:
A Web site can have more than one skip link that target different locations in a document. We could reasonably assume that the first, and if only, skip link is one that targets a `#main` element.

```suggestion
interface Props {
    label?: string;
    class?: string;
    href?: string;
}

const { label = 'Skip to main content', href = '#main', class: propsClass, ...rest } = Astro.props;

const classes = ['c-skip-link', propsClass];
---

<c-skip-link class:list={classes} {...rest} data-no-swup>
    <Button href={href}>{label}</Button>
</c-skip-link>
```